### ISSUE TITLE
GC is running

### ISSUE THREAD

**Issue by KNWBIN** [goharbor/harbor#13512]
When harbor is running gc, I restarted harbor.Now this gc has always been running and there is no way to cancel it.This also causes the harbor to be always read-only

version： 
Harbor
Version v1.8.1-16b59c41

![image](https://user-images.githubusercontent.com/42708020/99188585-9a778080-2797-11eb-8b6a-8fe7012937cf.png)

**Comment by wy65701436** [comment]
The GC in Harbor v1.8.1, it is read-only mode, and the execution time depends on how many blobs(GC candidates) do you have and the storage kind you used. Cannot stop is by designed as it may break the distribution storage.

Highly recommend to upgrade your Harbor to latest(v2.1) as the GC is changed to non-blocking mode, you can still pulling/pushing during GC.

**Comment by KNWBIN** [comment]
At first I wanted to modify harbor-db. After reviewing some articles, I found that although the status of GC is running, it does not affect the use of harbor. It is possible to execute SQL shutdown in harbor-db. The reason for the read-only is that after I executed the GC, the read-only in the configuration was turned on. When the new GC was executed, the read-only configuration was still turned on. After I manually turned it off, it was normal.
